app/jobs/reports/base_report.rb

```diff
@@ -7,6 +7,20 @@ class BaseReport < ApplicationJob
     # We use good_job's concurrency features to cancel "extra" or duplicative runs of the same job
     discard_on GoodJob::ActiveJobExtensions::Concurrency::ConcurrencyExceededError
 
+    def self.transaction_with_timeout(rails_env = Rails.env)
```

Comment:
rubocop got mad this was after the `private` keyword because static methods are public